python/tk_3dsmax/menu_generation_2025.py

```diff
@@ -0,0 +1,187 @@
+# Copyright (c) 2024 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+"""
+Menu handling for 3ds Max
+"""
+from pymxs import runtime as rt
+
+from .menu_generation import MenuGenerator, AppCommand
+
+MENU_LABEL = "Flow Production Tracking"
+HELPMENU_ID = "cee8f758-2199-411b-81e7-d3ff4a80d143"
```

Comment:
I saw this in plugins/basic/python/tk_3dsmaxplus_basic/constants.py already. Can we avoid the duplication?

Reply:
they're running in different scopes, but I make a single source of truth.